dashboard/static/css/grades.css

```diff
@@ -0,0 +1,119 @@
+body {
+    font: 10px sans-serif;
+}
+
+.axis path,
+.axis line {
+    fill: none;
+    stroke: #000;
+    shape-rendering: crispEdges;
+}
+
+.filter {
+    color: #fff
+}
+
+.B {
```

Comment:
Do we have any CSS naming rules? Maybe this . should be `grade--B`based on BEM rules?  We also should probably get some SASS library here here to make these easier to configure in the future. Maybe another ticket.